src/feature/dupe-check.ts

```diff
@@ -91,18 +91,29 @@ export async function checkForDuplicates() {
     .join(', ')
     .replace(/, ([^,]*)$/, ' and $1');
 
+  // The 'Duplicate of' keyword only works if its only it in the comment.
```

Comment:
Do we really need the timeline event though? I'm unaware of it actually doing anything beyond just showing up as a special item on the issue.

Reply:
On a second thought no, I first thought about it because I was planning to use the issue timeline to check if it was closed because it was a duplicate, but since the reopening got removed, I can rollback this too 👍🏻

Reply:
We could automatically add a label for duplicates. I'm not sure what your use case for seeing it is though.

Reply:
No, there's no need anymore now that the reopening logic got removed. I will remove the double comment part of the code tomorrow 👍🏻